README.md

```diff
@@ -880,10 +880,22 @@ $ ronin new payload my_payload.rb
 
 ### Archive
 
-Archive a files:
+Archive files:
 
 ```shell
-$ ronin archive foo.txt bar.txt
+$ ronin archive arch1.txt arch2.txt
+```
+
+Archive files to the specified file:
+
+```shell
+$ ronin archive -o archived.zip arch1.txt arch2.txt
+```
+
+Archive files using tar format:
+
+```shell
+$ ronin archive -f tar arch1.txt arch2.txt
```

Comment:
Maybe defaulting the output file based on the first file's name, might be a bit confusing to new users? Maybe requiring `options[:output]` might be better (ex: `ronin archive -o output.tar file1.txt file2.txt`)? Or we could require that the first argument be the output file (ex: `ronin archive output.tar file1.txt file2.txt`).